peekingduck/mp_runner.py

```diff
@@ -0,0 +1,94 @@
+# Copyright 2021 AI Singapore
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Prototype engine to run individual peekingduck nodes in multiple processes
+"""
+
+
+import logging
+import importlib
+
+from typing import Dict, List, Callable
+from multiprocess import Process, Manager  # pylint: disable=no-name-in-module
```

Comment:
```suggestion
from multiprocess import Process, Manager  # pylint: disable=no-name-in-module, import-error
```

The multiprocess is causes issues when run with the linter. recommend disabling lint.

Reply:
Oh, this problem may arise because multiprocess is not in the requirement.txt, i can add it to the requirements.txt for this branch